README.md

```diff
@@ -0,0 +1,26 @@
+![SPIRE Logo](/doc/images/spire_logo.png)
+
+This contains the service definitions and code generated stubs for public SPIRE
+APIs.
+
+
+## Versioning
+
+This repository is tagged along with SPIRE releases with the same name, even if
+there are no changes to the APIs between SPIRE versions. This allows consumers
+to always pick a tag that matches up with their deployment. Even so, SPIRE
+maintains API compatibility between SPIRE versions. SPIRE will clearly indicate
+in the CHANGELOG when APIs are deprecated and issue warnings at runtime when
+they are used well in advance of any removal.
+
+## Migrating Code
+
+To migrate existing code that consumed SPIRE service definitions from the
+github.com/spiffe/spire/proto/spire Go module, see
+[MIGRATING](/MIGRATING.md).
+
+## Contributing
+
+To contribute to this repository, see [CONTRIBUTING](/CONTRIBUTING.md).
```

Comment:
I think that it would be good to have this section similar to what we have in the SPIRE repo, where we also describe who maintains the project and we reference to the governance policies.